conda_forge_tick/mamba_solver.py

```diff
@@ -282,8 +284,9 @@ def is_recipe_solvable(feedstock_dir):
         solvable = solvable and _solvable
         cbc_name = os.path.basename(cbc_fname).rsplit(".", maxsplit=1)[0]
         errors.extend([f"{cbc_name}: {e}" for e in _errors])
+        solvable_by_cbc[cbc_name] = _solvable
 
-    return solvable, errors
+    return solvable, errors, solvable_by_cbc
```

Comment:
There are two branches above that return early. They need a third return argument too.